src/main/java/com/indeed/jiraactions/api/ApiCaller.java

```diff
@@ -3,178 +3,130 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.indeed.jiraactions.JiraActionsIndexBuilderConfig;
+
+import okhttp3.Cookie;
+import okhttp3.CookieJar;
+import okhttp3.Headers;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
 import org.apache.commons.codec.binary.Base64;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.net.ssl.HttpsURLConnection;
-import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.Objects;
 
 public class ApiCaller {
     protected final JiraActionsIndexBuilderConfig config;
 
-    private static final Logger log = LoggerFactory.getLogger(ApiCaller.class);
+    private static final Logger log = LoggerFactory.getLogger(IssuesAPICaller.class);
     private static final ObjectMapper objectMapper = new ObjectMapper();
+    private static final CookieJar cookieJar = setupCookieJar();
```

Comment:
I'm not sure it matters, but why is this static?

Reply:
I originally had CookieJar as a static class and just left it in. I guess we can just call setupCookieJar() in the client builder now.

Reply:
CookieJar as a static class makes sense. But being a static variable means that if there were to be multiple instances of the ApiCaller class, they would share a single CookieJar. This isn't a good idea. Note that the CookieJar you've built isn't thread-safe, so if we were ever to make calls in parallel, that would be a problem.